content/module-reference/utility-modules/shared/tagging.md

```diff
@@ -207,11 +207,39 @@ It is possible to completely remove a tag from all images (whether selected or n
 - right-click on a tag in the _tag dictionary_ window, to show a pop-up menu, and choose "delete tag".
 - right-click on a branch node in the _tag dictionary_ window, to show a pop-up menu, and choose "delete node" to delete the selected node together with any child nodes.
 
-## import / export
+## import…
 
-The “import” button allows you to choose a text file (which must comply with the Lightroom tag text file format) and import its content. If a tag in the imported file already exists, its properties will be updated, otherwise a new tag will be created.
+The “import” button allows you to choose a text file and import its content. If a tag in the imported file already exists, its properties will be updated, otherwise a new tag will be created.
 
-The “export” button exports your entire tag dictionary into a text file of your choice (Lightroom tag text file format).
+The import parser creates a separate tag for _every line_ in the file. A line enclosed in square brackets `[]` will be stored as a __category__ in darktable. Two basic line formats are understood:
+
+darktable tag format
+: The tag hierarchy is written in a single line where nodes are separated by the pipe symbol.
+
+```
+places|Austria|Graz
+places|Austria|Vienna
+[places|Austria]
+```
+
+Lightroom tag text file format
+: The tag hierarchy is written in multiple lines where each node is indented further by tabs.
+
+```
+places
+    [Austria]
+        Graz
+places
+    Austria
+        Vienna
+```
+In both examples four tags are created: `places`, `places|Austria`, `places|Austria|Graz` and `places|Austria|Vienna`. In the first example `places|Austria` is set as a category.
+
+Note: if the second example is imported after the first one, `places|Austria` remains a category.
```

Comment:
I've added this note just for your understanding. Not sure that's necessary at this level of doc.